nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -173,6 +173,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The `cudaPackages` package scope has been updated to `cudaPackages_12`.
 
+- Ada packages (libraries and tools) are now in the gnatPackages scope. gnatPackages uses the default gnat compiler, gnat12Packackages and gnat13Packages uses the respective compiler version. Gnatprove is build in the version of the compiler.
```

Comment:
Minor editing I'd do:

```suggestion
- Ada packages (libraries and tools) have been moved into the `gnatPackages` scope. `gnatPackages` uses the default GNAT compiler, `gnat12Packackages` and `gnat13Packages` use the respective matching compiler version.
```

For spark2014 I'd write something like »`spark2014` has been renamed to `gnatprove`. A version of `gnatprove` matching different GNAT versions is available from the different `gnatPackages` sets.«

Reply:
Done